src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -55,16 +56,17 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
    */
   public List<Message> getMessages(String user) {
+    List<Message> messages = new ArrayList<>();
```

Comment:
Actually I meant like maybe removing the " = new ArrayList<>()" part might be better because intellj somehow flags it as code redundancy...but its okay I can remove it too after you merge it in haha

Reply:
Here's the screenshot: 
![image](https://user-images.githubusercontent.com/26247692/58846892-6a2b4780-86b3-11e9-95b5-f52da457f0bd.png)

Reply:
Here's the screenshot:
![image](https://user-images.githubusercontent.com/26247692/58846977-c9895780-86b3-11e9-902c-1d06a5c084de.png)